clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.h

```diff
@@ -1094,6 +1094,15 @@ class CIRToLLVMGetBitfieldOpLowering
                   mlir::ConversionPatternRewriter &) const override;
 };
 
+class CIRToLLVMDeleteArrayOpLowering
+  : public mlir::OpConversionPattern<cir::DeleteArrayOp> {
+public:
+  using mlir::OpConversionPattern<cir::DeleteArrayOp>::OpConversionPattern;
+
+  mlir::LogicalResult
+  matchAndRewrite(cir::DeleteArrayOp op, OpAdaptor adaptor, mlir::ConversionPatternRewriter &rewriter) const override;
```

Comment:
This violates 80-col, as you gonna see github complaining (make sure you run clang-format in your diffs before putting the PR up).

Reply:
Ran clang-format. Will keep in mind.